llama-index-integrations/llms/llama-index-llms-ibm/llama_index/llms/ibm/base.py

```diff
@@ -335,6 +365,12 @@ def complete(
             raw=response,
         )
 
+    @llm_completion_callback()
+    async def acomplete(
+        self, prompt: str, formatted: bool = False, **kwargs: Any
+    ) -> CompletionResponse:
+        return self.complete(prompt, formatted=formatted, **kwargs)
```

Comment:
there's no async client or async client methods?

Reply:
Part of them are still under construction in the source package, so they will be delivered soon (as the next step).

Reply:
Great!